Tests/TuistAutomationTests/Utilities/AppBundleLoaderTests.swift

```diff
@@ -78,6 +78,33 @@ final class AppBundleLoaderTests: TuistUnitTestCase {
         )
     }
 
+    // TODO: Ask for signed tvOS app
+    func test_load_appletv_app_bundle() async throws {
```

Comment:
I wouldn't try to decode the whole bundle when the issue is the `Info.plist`. Adding a full bundle is costly from the maintenance perspective and it increases the size of our git repo.

I'd suggest following [this](https://github.com/tuist/tuist/pull/7498/files#diff-62cae9a44d45cc74e3e5bd7450198069edd5f6142e4aa3f4abc75cd7b86530f0R119) test and only test the decoding of the `Info.plist`